tests/test_itemized.py

```diff
@@ -111,24 +111,27 @@ def test_filter_sched_a_zip(self):
         [
             factories.ScheduleAFactory(contributor_zip=96789),
             factories.ScheduleAFactory(contributor_zip=9678912),
-            factories.ScheduleAFactory(contributor_zip=967891234)
+            factories.ScheduleAFactory(contributor_zip=967891234),
+            factories.ScheduleAFactory(contributor_zip='M4C 1M7')
         ]
-
-        results = self._results(api.url_for(ScheduleAView, contributor_zip=96789, **self.kwargs))
+        results = self._results(api.url_for(ScheduleAView, contributor_zip=967893405, **self.kwargs))
         self.assertEqual(len(results), 3)
     
-    def test_invalid_sched_a_zip(self): 
-        response = self.app.get(api.url_for(ScheduleAView,contributor_zip=9678912,cycle=2018))
-        self.assertEqual(response.status_code,400)
+        results = self._results(api.url_for(ScheduleAView, contributor_zip='M4C 1M55', **self.kwargs))
+        self.assertEqual(len(results), 1)
```

Comment:
Are we sure this is intended? 'M4C 1M55' will match 'M4C 1M7'.

Reply:
yes for now since we would rather be over-inclusive than under-inclusive